content/working_with/console/widgets/executions.md

```diff
@@ -24,11 +36,24 @@ In the actions menu on the right side of the execution row (click ![List icon](
  
  For details about cancelling executions, see [cancelling workflow executions]({{< relref "working_with/workflows/cancelling-execution.md" >}})
 
-![executions]( /images/ui/widgets/executions.png )
 
-After selecting an execution by clicking its row in the table a corresponding task graph is displayed.
+### Execution task graph
+
+Workflows are a set of tasks that can be executed on a node or a group of nodes. Most of them are implemented with tasks graph. When the workflow is being executed, the state of the tasks graph, and each operation in it, is persisted to storage. When the workflow is resumed, the tasks graph is reconstructed, and the execution continues.
```

Comment:
```suggestion
Workflows are tasks that can be executed on a node or a group of nodes. Most of them are implemented with a tasks graph. When the workflow is being executed, the state of the tasks graph, and each operation in it, is persisted to storage. When the workflow is resumed, the tasks graph is reconstructed, and the execution continues.
```

Reply:
I rephrased it a bit as I think "Workflows are tasks" is a bit misleading. What I wanted to describe here is that workflows contains tasks, then tasks can be visualized as tasks graph.